### ISSUE TITLE
BlockingIOCometSupport is used for Jetty 9.3.15

### ISSUE THREAD

**Issue by Artur-** [Atmosphere/atmosphere#2242]
Jetty 9.3.15 has changed how a WebSocketServerFactory is created: 
https://github.com/eclipse/jetty.project/commit/70247d74d943e178d91e87f5dd2959a0b927bcbe#diff-e23998b9c2005afca68b4a3e805163feR104

Therefore you get
```
16:30:52.559 ERROR [main] o.a.c.DefaultAsyncSupportResolver [DefaultAsyncSupportResolver.java:229] Failed to create comet support class: class org.atmosphere.container.Jetty9AsyncSupportWithWebSocket, error: null
16:30:52.559 ERROR [main] o.a.c.DefaultAsyncSupportResolver [DefaultAsyncSupportResolver.java:230] Switching to BlockingIO
```

**Comment by jfarcand** [comment]
@Artur- Might want to enable jsr356 by default to avoid all those problem with Jetty-keep-changing-public-api

**Comment by jfarcand** [comment]
Fixed by #2270
